extensions/ext/GL_EXT_maximal_reconvergence.txt

```diff
@@ -0,0 +1,109 @@
+Name
+
+  EXT_maximal_reconvergence
+
+Name Strings
+
+  GL_EXT_maximal_reconvergence
+
+Contact
+
+  Alan Baker, Google LLC (alanbaker 'at' google.com)
+
+Contributors
+
+  Alan Baker, Google LLC
+
+Status
+
+  Draft.
+
+Version
+
+  Last Modified Date:   December 14, 2023
+  Revision: 1
+
+Number
+
+  TBD.
+
+Dependencies
+
+  This extension can be applied to OpenGL GLSL versions 1.40
+  (#version 140) and higher.
+
+  This extension can be applied to OpenGL ES ESSL versions 3.10
+  (#version 310) and higher.
+
+  This extension is written against the OpenGL Shading Language
+  Specification, version 4.60 (revision 7), dated July 10, 2019.
+
+  Interacts with GL_KHR_vulkan_glsl.
+
+  Interacts with GL_EXT_control_flow_attributes.
+
+Overview
+
+  This extension adds the ability to annotate entry points via an attribute to
+  indicate that subgroups should reconverge in the same way as invocation
```

Comment:
Is this leftover from subgroup_uniform_control_flow?

Reply:
I think so.